workbench-core/accounts/src/services/costCenterService.test.ts

```diff
@@ -56,7 +55,7 @@ describe('CostCenterService', () => {
       environmentInstanceFiles: '',
       stackName: `${process.env.STACK_NAME!}-hosting-account`,
       status: 'CURRENT',
-      awsAccountId: 'awsAccountId',
+      awsAccountId: '123456789012',
```

Comment:
Instead of copy and pasting this, we should turn it into a const so it only needs to be updated in one place if the value changes.